MultiServer.py

```diff
@@ -2536,6 +2536,25 @@ async def main(args: argparse.Namespace):
             logging.info("No file selected. Exiting.")
             import sys
             sys.exit(1)
+        elif not args.disable_save:
```

Comment:
this code only gets run when selecting a multidata via file browser (rather than "open with"). This seems wrong to me and should probably be indented 1 level less?